supertokens_python/recipe/emailverification/utils.py

```diff
@@ -57,42 +64,49 @@ class ParentRecipeEmailVerificationConfig:
     def __init__(self,
                  get_email_for_user_id: Callable[[str, Dict[str, Any]], Awaitable[str]],
                  override: Union[OverrideConfig, None] = None,
-                 get_email_verification_url: Union[Callable[[
-                     User, Dict[str, Any]], Awaitable[str]], None] = None,
-                 create_and_send_custom_email: Union[Callable[[
-                     User, str, Dict[str, Any]], Awaitable[None]], None] = None
+                 get_email_verification_url: Union[Callable[[User, Dict[str, Any]], Awaitable[str]], None] = None,
+                 create_and_send_custom_email: Union[Callable[[User, str, Dict[str, Any]], Awaitable[None]], None] = None,
+                 email_delivery: Union[TypeInput[TypeEmailVerificationEmailDeliveryInput], None] = None
                  ):
         self.override = override
         self.get_email_verification_url = get_email_verification_url
         self.create_and_send_custom_email = create_and_send_custom_email
         self.get_email_for_user_id = get_email_for_user_id
+        self.email_delivery = email_delivery
 
 
 class EmailVerificationConfig:
     def __init__(self,
                  override: OverrideConfig,
                  get_email_verification_url: Callable[[User, Dict[str, Any]], Awaitable[str]],
-                 create_and_send_custom_email: Callable[[User, str, Dict[str, Any]], Awaitable[None]],
-                 get_email_for_user_id: Callable[[str, Dict[str, Any]], Awaitable[str]]
+                 get_email_for_user_id: Callable[[str, Dict[str, Any]], Awaitable[str]],
+                 email_delivery: Union[TypeInput[TypeEmailVerificationEmailDeliveryInput], None],
+                 email_service: Union[EmailDeliveryInterface[TypeEmailVerificationEmailDeliveryInput], None]
```

Comment:
this should just be one of them and it it's type should be `Callable[[All things needed by backward compatible service], TypeInput[TypeEmailVerificationEmailDeliveryInput]]`

Reply:
Fixed.